mod/nodeinfo.php

```diff
@@ -141,6 +141,8 @@ function nodeinfo_init(App $a) {
 		if (Addon::isEnabled('twitter')) {
 			$nodeinfo['metadata']['services']['inbound'][] = 'twitter';
 		}
+
+		$nodeinfo['metadata']['explicitContent'] = Config::get('system', 'explicit_content', False) == True;
```

Comment:
Code Standards: boolean values should be written lowercase except if the prevailing surrounding spelling is different,